bundles/AdminBundle/Resources/public/js/pimcore/document/editables/image.js

```diff
@@ -234,6 +242,7 @@ pimcore.document.editables.image = Class.create(pimcore.document.editable, {
                     this.datax.id = data["id"];
 
                     this.updateImage();
+                    this.checkValue();
```

Comment:
```suggestion
                    this.checkValue(true);
```
image editable should reset error border on upload